.github/workflows/pr-comment.yml

```diff
@@ -3,16 +3,13 @@ on:
   issue_comment:                                     
     types: [created, edited, deleted]
 jobs:
- job01:
-   if: contains(github.event.comment.html_url, '/pull/')    # check if the comments come from pull request, exclude those from issue.
-   runs-on: [re-centos7]
```

Comment:
This is a new feature. However, introducing it violates the agreed upon freeze period. Please be mindful of the established guidelines in the future.